acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/comparison/CompareSignMessageAcceptanceTest.java

```diff
@@ -33,6 +33,9 @@
 
 public class CompareSignMessageAcceptanceTest extends CompareApisAcceptanceTestBase {
 
+  private static final ObjectMapper OBJECT_MAPPER =
```

Comment:
this can be removed if you expose ObjectMapper from LOTUS_NODE instance.

Reply:
THINK I'm happy enough with this being here ...I can be persuaded if you push hard though

Reply:
sure thing ... it was a nit pick, so its OK to be as it is.